.codeclimate.yml

```
@@ -0,0 +1,17 @@
+plugins:
+  # https://docs.codeclimate.com/docs/git-legal
+  git-legal:
+    enabled: true
+      allow_strong_copyleft: true
```

```suggestion
    allow_strong_copyleft: true
```
This should be what causes the build to fail